### DISCUSSION TITLE
能否给计时添加上“备注”功能

### DISCUSSION THREAD

**Discussion by liza-li** [shion-app/shion#66] [Ideas]
首先非常感谢作者，Shion非常符合我的需求！
现在我在用“记录”这个功能来追踪我每天对计划的执行情况。目前，我下载的版本支持“计划”-“标签”两个层次的计时分类，能不能进一步开发“备注”这个功能，便于我完成一个时间块后，能够给这个时间块备注一些详细的文本信息？
功能类似于WorkingHours的“详细信息”
![Snipaste_2024-12-31_16-34-15](https://github.com/user-attachments/assets/fd83521b-89d2-4ef7-b4b8-04f4f3c0a060)

**Comment by hanaTsuk1** [comment]
没这个打算

文本记录已经有了，只不过不是和计时绑定在一起的

详情查看[这里](https://shion.app/zh/guide/record#obsidian)
